<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormOptPic</class>
 <widget class="QWidget" name="FormOptPic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pictures Options</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Trialmaps</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxSizeTrial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Size Pictures</string>
     </property>
     <widget class="QLineEdit" name="widthTrialLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>576</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="widthTriallabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Width (Px)</string>
      </property>
     </widget>
     <widget class="QLabel" name="heigthTrialLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Heigth (Px)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="heigthTrialLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>432</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxInterpo">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>141</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Interpolation Factor</string>
     </property>
     <widget class="QLineEdit" name="horizInterpoLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="horizInterpoLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Horizontal</string>
      </property>
     </widget>
     <widget class="QLabel" name="vertiInterpoLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertical</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="vertiInterpoLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabElecPlot">
    <attribute name="title">
     <string>ElecPlot</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxSizePlot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Size Pictures</string>
     </property>
     <widget class="QLineEdit" name="widthPlotLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1920</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="widthPlotLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Width (Px)</string>
      </property>
     </widget>
     <widget class="QLabel" name="heigthPlotLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Heigth (Px)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="heigthPlotLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1137</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Validate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
